pkg/systemd/services.css

```diff
@@ -1,5 +1,5 @@
 #services-page {
-  verflow-y: scroll;
+    overflow-y: scroll;
```

Comment:
There are two commits "Always show scroll bar". Squashing makes sense IMHO, as they read pretty well exactly the same.

Reply:
Of course, I did typo while rebasing and garrett caught it. But I thought he squashed it. Will squash it.